src/adapters/MeetingsJSONAdapter.js

```diff
@@ -92,13 +92,27 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
       observer.complete();
     });
 
-    // Attach local media stream if meeting `localVideo` property is not `null`.
+    // Attach media stream if meeting any of media stream property is not `null`.
     // We can not attach the MediaStream object in a JSON module, the work needs
     // to be done here.
-    const getMeetingWithLocalMedia$ = getMeeting$.pipe(
+    const getMeetingWithMedias$ = getMeeting$.pipe(
+      /* eslint-disable no-confusing-arrow */
       flatMap((meeting) =>
-        from(this.getLocalVideo()).pipe(map((localVideo) => (meeting.localVideo ? {...meeting, localVideo} : meeting)))
+        meeting.localVideo
+          ? from(this.getStream({video: true, audio: false})).pipe(map((localVideo) => ({...meeting, localVideo})))
+          : of(meeting)
+      ),
+      flatMap((meeting) =>
+        meeting.remoteVideo
+          ? from(this.getStream({video: true, audio: false})).pipe(map((remoteVideo) => ({...meeting, remoteVideo})))
+          : of(meeting)
+      ),
+      flatMap((meeting) =>
```

Comment:
How can you have the case where `remoteAudio` and `remoteVideo are sent together?

Reply:
the media keeps getting attached. that's why I kept it this way.

Reply:
or should I say **KEEP**